v0/destinations/ga/transform.js

```diff
@@ -720,6 +720,38 @@ function processEComGenericEvent(message, destination) {
         throw new Error("unknown TransactionEvent type");
     }
   }
+  const { products } = message.properties;
```

Comment:
we seem to be adding this product properties logic in few other places.
Better to move this as a separate method?

Reply:
done please check.